src/js/classes/Table.js

```diff
@@ -347,12 +347,16 @@ export class RankedTable {
   sort(initialSort) {
     if (!initialSort) this.header.clearedSortedCells();
 
-    // data doesn't have rank or spacer, so subtract two from the index
+    // data doesn't have rank, so subtract one from the index
     const dataCol = this.sortCol - 1;
     this.data.sort((a, b) => {
-      if (a[dataCol] < b[dataCol]) {
+      // Assumes that we only want to sort numbers, which is fine for now
+      // May need to support sorting multiple types
+      const i = Number(a[dataCol]);
+      const j = Number(b[dataCol]);
```

Comment:
I'm casting the "+12" to a number because if you don't it I think it does lexicographical sort.

I considered having a "SignedNumberCell" class and then just injecting the "+" or "-" on render but it seemed like overkill for this case. May still do it if it's cleaner.

Reply:
Ah, forgot about that column. Good thinking!